structure/tree/avl_test.go

```diff
@@ -263,14 +263,14 @@ func TestGet(t *testing.T) {
 	Insert(&root, 3)
 
 	n := Get(root, 4)
-	if n.Key != 4 {
+	if n.key != 4 {
 		t.Error("key should be 4")
 	}
-	if n.Right.Key != 5 {
+	if n.right.key != 5 {
```

Comment:
I'm talking about this. You said that every member is exported but you are only testing is not exported.

Reply:
Sorry about that. I'm used to make each field of struct un-exported. I will make it exported as soon as possible.

Reply:
Haha, I know. I have the same habit 😂

Reply:
Exactly. I need to pay more attention to this.